app/views/shared/_breadcrumbs.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class='al-search-breadcrumb'>
+<div class='al-search-breadcrumb' role='navigation' aria-label='breadcrumb'>
```

Comment:
Should the `aria-label` value be stored in the i18n translations files?

Reply:
yes, it should be. thanks for catching that.